info/guide/guide-chapter-3.md

```diff
@@ -396,6 +471,62 @@ if (( x >  100) ||
     y < 100 && !isFoo()) {}
 ```
 
+`String template` also can be split in white space in string text
+
+**Invalid example:**
+```kotlin
+val nameString = "This is very long string template"
+```
+
+**Valid example:**
+```kotlin
+val nameString = "This is very long" +
+        " string template"
+```
+
+Long `Lambda argument` should be split:
+
+**Invalid example:**
+```kotlin
+val variable = a?.filter { it.elementType == true } ?: null
+```
+
+**Valid example:**
+```kotlin
+val variable = a?.filter {
+    it.elementType == true
+} ?: null
+```
+
+Long one line `When Entry` should be split:
```

Comment:
When used as an adjective, "one line" should be hyphen-separated: "one-line". 

"Single-line" is even more clear.